### PR TITLE
Announce KubeDB v2022.05.24

### PR DESCRIPTION
**Pull request by kamolhasan**
Signed-off-by: kamolhasan <kamol@appscode.com>

### CODE REVIEW

File: content/post/kubedb-v2022.05.24/index.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  - mysql
+  - postgresql
+  - redis
+  - kubedb
+  - elasticsearch-dashboard
+  - kibana
+  - schema-manager
+---
+
+We are pleased to announce the release of [KubeDB v2022.05.24](https://kubedb.com/docs/v2022.05.24/setup/). This post lists all the major changes done in this release since the last release. This release offers some major features like **MySQL Semi-Synchronous Replication**, **MongoDB Arbiter**, **PGBouncer**, **MariaDB Schema Manager**, **ProxySQL**, **Redigned Redis**, **Elasticsearch V8**, **OpenSearch Dashboard**, etc. It also contains various improvments and bug fixes. You can find the detailed changelogs [here](https://github.com/kubedb/CHANGELOG/blob/master/releases/v2022.05.24/README.md).
```
Comment (rakibulhossain) [inline content/post/kubedb-v2022.05.24/index.md:24]: "Redesigned Redis"

File: content/post/kubedb-v2022.05.24/index.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+     - ReadWriteOnce
+   resources:
+     requests:
+       storage: 10Gi
+ terminationPolicy: WipeOut
+```
+
+**What's New in MYSQL CRO?**
+
+In the `spec.topology` you refer to `SemiSync` Mode and in the `spec.topolgy.semiSync` you can refer to how many replicas, the source will wait for an acknowledgment before committing, how long the source will wait for replicas before going back to asynchronous replication, and the errant transaction recovery policy. We support two recovery policies for errant transactions that is PseudoTranscation and Clone.
```
Comment (rakibulhossain) [inline content/post/kubedb-v2022.05.24/index.md:60]: `spec.topolgy.semiSync` -> `spec.topology.semiSync`

File: content/post/kubedb-v2022.05.24/index.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+      requests:
+        storage: 1Gi
+  internalUsers:
+      kibana_system:
+        secretName: kibana-system-custom-cred
+```
+
+## Dashboard
+
+In this release, we are expanding our support for dashboards. Now KubeDB supports `opensearch-dashboard` with ElasticsearchVersion CRO `opensearch-1.1.0`, `opensearch-1.2.0`, and `opensearch-1.3.2`. It also supports kibana  with ElasticsearchVersion CRO `xpack-8.2.0`, `xpack-7.17.3`, `xpack-7.16.2`, `xpack-7.13.2`, `xpack-7.12.0`, `xpack-7.9.1`, and `xpack-6.8.22`.
```
Comment (raihankhan) [inline content/post/kubedb-v2022.05.24/index.md:235]: Typo -> OpenSearch Dashboards

**Review by rakibulhossain — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2022.05.24/index.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+   defaultTTL: "10m"
+ deletionPolicy: "Delete"
+```
+
+The given YAML will create a MariaDB database schema in the cluster which will eventually end up creating a database demo inside  mariadb-server which is a KubeDB MariaDB server. The demo database will be created with characterset `utf8` and `encryption disabled`  as it is mentioned in the `spec.database.configuration` section. The operator will create a user and its credentials using the `KubeVault` and bind the permission under the subject which is mentioned in the `spec.accessPolicy.subjects` section. The credential and permission will be valid for 10 minutes. The time duration is  mentioned in the `spec.accessPolicy.defaultTTL` section. After that, the schema will be expired.
+
+## Elasticsearch
+
+In this release, We are happy to introduce the support for `Elasticsearch V8`. Now, you can provision and manage Elasticsearch V8 with KubeDB. New supported ElasticsearchVersion CROs are `xpack-8.2.0`, `kubedb-xpack-8.2.0`, and `opensearch-1.3.2`. The `kubedb-xpack-8.2.0` packed with  `elasticsearch:8.2.0` with pre-installed plugins; `repository-s3`, `repository-azure`, `repository-hdfs`, and `repository-gcs`.
```
Comment (raihankhan) [inline content/post/kubedb-v2022.05.24/index.md:209]: The `kubedb-xpack-8.2.0` is packed with  `elasticsearch:8.2.0` with pre-installed plugins: `repository-s3`, `repository-azure`, `repository-hdfs`, and `repository-gcs`.

File: content/post/kubedb-v2022.05.24/index.md (hunk 3 above)
Comment (raihankhan) [inline content/post/kubedb-v2022.05.24/index.md:235]: Typo: `opensearch-dashboard` > `OpenSearch Dashboards`

**Review by raihankhan — CHANGES_REQUESTED** [review]

**Review by ArnobKumarSaha — DISMISSED** [review]

**Review by raihankhan — DISMISSED** [review]
LGTM!

**Review by TasdidurRahman — APPROVED** [review]
LGTM

**Review by Shaad7 — COMMENTED** [review]
LGTM

**Review by Shaad7 — APPROVED** [review]

**Review by rakibulhossain — APPROVED** [review]

**Review by rakibulhossain — DISMISSED** [review]
LGTM!

**Review by spectro30 — DISMISSED** [review]
LGTM

**Review by Emon46 — DISMISSED** [review]
LGTM!

**Review by Emon46 — APPROVED** [review]
LGTM!

**Pull request merged**